RFC0000-RFC-Process.md

```diff
@@ -62,59 +62,53 @@ Description and rationale.
+* New comments are not being sought.
+* No one has begun implementing the RFC, and there are no current plans to implement the RFC.
+* The community is invited to implement the RFC (unless explicitly stated in the RFC).
 
-At this point, the community discusses the RFC.
+### Experimental
 
-After the `Comments Due` date, the Author summarizes the comments, submits a new Pull Request to make the final RFC Draft, and asks the PowerShell Committee to vote.
+RFCs in the `Experimental` stage have been accepted by the Committee, and code is being written to provide a working example of the proposed design change to get additional feedback.
 
-Just before voting, the PowerShell Comittee merges the Pull Request.
-The PowerShell Committee then votes to accept or reject the RFC Draft.
+### Experimental-Accepted
 
-* Draft-Accepted
-
-The PowerShell Committee has reviewed the RFC Draft and comments, and has voted to accept the RFC Draft.
-
-At this point, new comments are not being sought.
-
-No one has begun implementing the RFC, and there are no current plans to implement the RFC.
-
-The community is invited to implement the RFC.
-
-* Experimental
-
-Comments have been reviewed and code is being written to provide an working example of the proposed design change to get further feedback.
+Feedback from the experimental implementation and RFC have been reviewed.
 
-* Experimental-Accepted
+Because this working prototype already exists in preview builds available on GitHub, the community provide feedback on the implementation as issues in the [PowerShell/PowerShell repository](https://github.com/powershell/powershell) 
 
-Feedback from the experimental implementation and RFC have been reviewed.
-Engineering team will work towards final implementation in code to match the RFC.
+As the engineering team or code contributor works towards a final implementation, they should submit pull requests to PowerShell-RFC in order to keep the RFC in sync with the implementation.
+These pull requests shall be reviewed and accepted by the Committee, but a formal vote is not necessary.
+The Committee should merely confirm that the changes in the RFC match the working implementation.
 
-* Rejected
+### Rejected
 
-Based on community feedback, this RFC was decided to not proceed any further.
+RFCs in the `Rejected` state were rejected by the Committee who decided not to proceed further.
 
-* Withdrawn
+### Withdrawn
 
-Author has decided not to pursue this RFC any further
+RFCs in the `Withdrawn` state were rescinded by the RFC Author.
 
-* Final
+### Final
 
-Design and implementation is considered complete.
-Any proposed changes would be through a new RFC.
+RFCs in the `Final` state are considered fully complete and implemented in PowerShell.
+Any proposed changes should be made through a new RFC or via an Issue in the [PowerShell/PowerShell repository](https://github.com/powershell/powershell).
```

Comment:
I don't really want to get into the business of deciding what constitutes a "modification to an existing RFC" vs. "a fully new feature". It's a lot easier to just say "that's done now, next RFC". 

I could be wrong here, though. Do you guys have experience with this on either side at Chef?